bin/generate_lts_report.py

```diff
@@ -105,7 +107,22 @@ def get_build_report(project_url, unfinished=False,
 
     r = requests.get(template_url)
     r.raise_for_status()
-    return r.text
+    callback_url = r.json()["url"]
+    report = ""
+    for i in range(1, timeout):
+        r = requests.get(callback_url)
+        r.raise_for_status()
+        if r.json()['status_code'] is None:
+            time.sleep(1)
+            continue
```

Comment:
maybe print some message to stdout so it's clear what happens.

Reply:
This report either gets copy/pasted into an email, or personally I run from inside mutt/vim with :read !generate_lts_report.py 4.14. In both cases I would prefer to avoid output that shouldn't be in the final report. I tried putting it to stderr but vim still picks it up, and on a terminal it'd also be easy to accidentally include.